CHECKS.md

```diff
@@ -219,6 +226,51 @@ but a.short.domain.ca is four components deep.
 
 The `URL` directive should always come after the `Title` is a given stanza.
 
+---------
+
+### L1011 - `AddUserHeader` directive is out of order
```

Comment:
We should avoid having checks with the same title.  I think it would be easier to modify the L1011 title.  Something options (additions italicized):

- _Bare_ `AddUserHeader` directive is out of order
- _Ending_ `AddUserHeader` directive is out of order
- `AddUserHeader` directive _with no qualifiers_ is out of order